src/pages/[[...slug]].tsx

```diff
@@ -259,11 +259,24 @@ export async function getStaticProps(context: GetStaticPropsContext) {
       }
     }
     try {
-      const resource = await getStaticPropsResource(
-        resourceType,
-        pathInfo,
-        expandedContext
-      )
+      let resource
+      try {
+        resource = await getStaticPropsResource(
+          resourceType,
+          pathInfo,
+          expandedContext
+        )
+      } catch (e) {
+        const util = await import('util')
+        console.error('\n\nFailed to fetch resource:')
+        console.error(
+          util.inspect(
+            { resourceType, pathInfo, expandedContext },
+            { colors: true, depth: null }
+          )
+        )
+        throw e
+      }
```

Comment:
Another change not related to the fix, but it helped me diagnose the issue. I think this'll be really helpful for debugging later too.

Reply:
neat